src/app/contentful.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core'
+import * as contentful from 'contentful'
+import { Observable } from 'rxjs/Observable'
+import 'rxjs/add/observable/fromPromise'
+import 'rxjs/add/operator/map'
+
+@Injectable()
+export class ContentfulService {
+
+  private client
+
+  constructor () {
+    // Don't worry - these just give access to a read only query-able API - everything it has access to is public anyway
+    this.client = contentful.createClient({
+      space: 'yswja3mg62td',
+      accessToken: '3f04660d6e548431504836bafa5f3bd655a3a4581ad72a1e5f6c952b48efe7d8'
```

Comment:
This isn't a bad thing to expose is it?

Reply:
I'm a little conflicted tbh... It could be abused if someone wanted to use it to burn through our free api request limit... But without a real back-end there isnt really another option. I think since it's read only and the only attack would be a brute force to use up free api requests is fairly safe. If we noticed an attack we could spin up a back-end at that time.

Reply:
👍